batch_test.go

```diff
@@ -70,6 +71,7 @@ func TestBatchWrite(t *testing.T) {
 	}
 
 	testWrite := func(t *testing.T, db *DB) {
+		txCreateBucket(t, db, DataStructureBTree, bucket, nil)
 		key := func(i int) []byte {
```

Comment:
this function for getting the key can be replaced with the key generation function.